dojo/product_type/views.py

```diff
@@ -22,70 +24,99 @@
-                                 extra_tags='alert-success')
-            return HttpResponseRedirect(reverse('product_type'))
+            messages.add_message(
+                request,
+                messages.SUCCESS,
+                "Product type added successfully.",
+                extra_tags="alert-success",
+            )
+            return HttpResponseRedirect(reverse("product_type"))
     add_breadcrumb(title="Add Product Type", top_level=False, request=request)
-    return render(request, 'dojo/new_product_type.html', {
-        'name': 'Add Product Type',
-        'metric': False,
-        'user': request.user,
-        'form': form,
-    })
+    return render(
+        request,
+        "dojo/new_product_type.html",
+        {
+            "name": "Add Product Type",
+            "metric": False,
+            "user": request.user,
+            "form": form,
+        },
+    )
 
 
 @user_passes_test(lambda u: u.is_staff)
 def edit_product_type(request, ptid):
     pt = get_object_or_404(Product_Type, pk=ptid)
-    form = Product_TypeForm(instance=pt)
-    if request.method == 'POST':
-        form = Product_TypeForm(request.POST, instance=pt)
-        if form.is_valid():
-            pt = form.save()
-            messages.add_message(request,
-                                 messages.SUCCESS,
-                                 'Product type updated successfully.',
-                                 extra_tags='alert-success')
-            return HttpResponseRedirect(reverse('product_type'))
+    form1 = Product_TypeForm(instance=pt)
+    form2 = Delete_Product_TypeForm(instance=pt)
+    if request.method == "POST" and request.POST.get("edit_product_type"):
+        form1 = Product_TypeForm(request.POST, instance=pt)
+        if form1.is_valid():
+            pt = form1.save()
+            messages.add_message(
+                request,
+                messages.SUCCESS,
+                "Product type updated successfully.",
+                extra_tags="alert-success",
+            )
+            return HttpResponseRedirect(reverse("product_type"))
+    if request.method == "POST" and request.POST.get("delete_product_type"):
+        form2 = Delete_Product_TypeForm(request.POST, instance=pt)
+        if form2.is_valid():
+            pt.delete()
+            messages.add_message(
+                request,
+                messages.SUCCESS,
+                "Product type Deleted successfully.",
+                extra_tags="alert-success",
+            )
+            return HttpResponseRedirect(reverse("product_type"))
     add_breadcrumb(title="Edit Product Type", top_level=False, request=request)
-    return render(request, 'dojo/edit_product_type.html', {
-        'name': 'Edit Product Type',
-        'metric': False,
-        'user': request.user,
-        'form': form,
-        'pt': pt})
+    return render(
+        request,
+        "dojo/edit_product_type.html",
```

Comment:
As recommended by [PEP- 498](https://www.python.org/dev/peps/pep-0498/), let's keep single quotes, unless double quotes are required.
```suggestion
        'dojo/edit_product_type.html',
```
And following lines.